src/lib/netlist/macro/modules/nlmod_icl8038_dip.cpp

```diff
@@ -52,7 +52,7 @@ NETLIST_START(ICL8038_DIP)
 
 	ALIAS(11, VI.ON) // GND
 	ALIAS(9, V_SQR.OP) // Square out
-	ALIAS(3, VO.OP) // Triag out
+	ALIAS(3, VO.OP) // Trigonal out
```

Comment:
Should be triangle, not “trigonal”.

Reply:
Yes, please change it. And everything else you like to please as well - I am not a native speaker and appreciate the feedback.